From 20566c60d4977976cb31d654b173a41fc2437fa7 Mon Sep 17 00:00:00 2001
From: "Srivatsa S. Bhat (VMware)" <srivatsa@csail.mit.edu>
Date: Fri, 31 Mar 2023 14:31:12 -0700
Subject: [PATCH 2/2] linux-rt, Makefile: Add kernel flavor info to the
 generated version.h file

Add "PHOTON_KERNEL_FLAVOR_RT" macro definition to the generated file
include/generated/uapi/linux/version.h to identify Photon's linux-rt
kernel flavor.

Signed-off-by: Srivatsa S. Bhat (VMware) <srivatsa@csail.mit.edu>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 55f1c4f1c8c1..abe90a9cda07 100644
--- a/Makefile
+++ b/Makefile
@@ -1349,7 +1349,7 @@ define filechk_version.h
 	echo '#define PHOTON_KERNEL';                                                   \
 	echo '#define PHOTON_KERNEL_RELEASE                                             \
 	$(shell echo -n "$(KERNELRELEASE)" | cut -d'-' -f2 | cut -d'.' -f1)';           \
-	echo '#define PHOTON_KERNEL_FLAVOR_'
+	echo '#define PHOTON_KERNEL_FLAVOR_RT'
 endef
 
 $(version_h): PATCHLEVEL := $(or $(PATCHLEVEL), 0)
-- 
2.25.1

